crates/sui-graphql-rpc/src/main.rs

```diff
@@ -6,14 +6,33 @@ use std::path::PathBuf;
 
 use clap::Parser;
 use sui_graphql_rpc::commands::Command;
-use sui_graphql_rpc::config::{ConnectionConfig, ServerConfig, ServiceConfig};
-use sui_graphql_rpc::config::{Ide, TxExecFullNodeConfig};
+use sui_graphql_rpc::config::{
+    ConnectionConfig, Ide, ServerConfig, ServiceConfig, TxExecFullNodeConfig, Version,
+};
 use sui_graphql_rpc::server::builder::export_schema;
 use sui_graphql_rpc::server::graphiql_server::{
     start_graphiql_server, start_graphiql_server_from_cfg_path,
 };
 use tracing::error;
 
+// WARNING!!!
+//
+// Do not move or use similar logic to generate git revision information outside of a binary entry
+// point (e.g. main.rs). Placing the below logic into a library can result in unessesary builds.
```

Comment:
```suggestion
// point (e.g. main.rs). Placing the below logic into a library can result in unnecessary builds.
```

Reply:
Ha, I should fix this in the other places too.